### PR TITLE
Feature/epoint design

### PR DESCRIPTION
**Pull request by ue**

### CODE REVIEW

File: src/components/points/view/pointsView.js [hunk 1]
```diff
@@ -50,12 +50,14 @@ class PointsView extends Component {
      const {
        userActivities, userPoints, intl, isClaiming, claimPoints,
      } = this.props;
-     const isActiveIcon = true;
+     // TODO:
```
Comment (mistikk) [inline src/components/points/view/pointsView.js]: TODO ?

File: src/screens/points/screen/pointsScreen.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      <View style={styles.container}>
+        <Header />
+        {/* <NoPost
+          style={styles.container}
+          imageStyle={styles.image}
+          source={MESSAGES_IMAGE}
+          defaultText={intl.formatMessage({
+            id: 'messages.comingsoon',
+          })}
+        /> */}
```
Comment (mistikk) [inline src/screens/points/screen/pointsScreen.js]: Can you remove unnecessary comments

File: src/screens/points/screen/pointsScreen.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+        {isLoggedIn ? (
+          <Points />
+        ) : (
+          <NoPost
+            style={{ flex: 1 }}
```
Comment (mistikk) [inline src/screens/points/screen/pointsScreen.js]: can you move to style file

**Comment by ue** [comment]
@mistikk Fixed

**Pull request merged**